user_guide_src/source/libraries/email.rst

```diff
@@ -113,35 +113,37 @@ sending email.
                                         or ``smtp``
-**mailPath**        /usr/sbin/sendmail  None                         The server path to Sendmail.
-**SMTPHost**        No Default          None                         SMTP Server Hostname.
-**SMTPUser**        No Default          None                         SMTP Username.
-**SMTPPass**        No Default          None                         SMTP Password.
-**SMTPPort**        25                  None                         SMTP Port. (If set to ``465``, TLS will be used for the connection
+**$mailPath**       /usr/sbin/sendmail                               The server path to Sendmail.
+**$SMTPHost**                                                        SMTP Server Hostname.
+**$SMTPUser**                                                        SMTP Username.
+**$SMTPPass**                                                        SMTP Password.
+**$SMTPPort**       25                                               SMTP Port. (If set to ``465``, TLS will be used for the connection
                                                                      regardless of ``SMTPCrypto`` setting.)
-**SMTPTimeout**     5                   None                         SMTP Timeout (in seconds).
-**SMTPKeepAlive**   false               ``true``/``false`` (boolean) Enable persistent SMTP connections.
-**SMTPCrypto**      tls                 ``tls``, ``ssl``, or         SMTP Encryption. Setting this to ``ssl`` will create a secure
+**$SMTPTimeout**    5                                                SMTP Timeout (in seconds).
+**$SMTPKeepAlive**  false               ``true``/``false``           Enable persistent SMTP connections.
+**$SMTPCrypto**     tls                 ``tls``, ``ssl``, or         SMTP Encryption. Setting this to ``ssl`` will create a secure
                                         empty string (``''``)        channel to the server using SSL, and ``tls`` will issue a
                                                                      ``STARTTLS`` command to the server. Connection on port ``465`` should
                                                                      set this to an empty string (``''``). See also
                                                                      :ref:`email-ssl-tls-for-smtp`.
-**wordWrap**        true                ``true``/``false`` (boolean) Enable word-wrap.
-**wrapChars**       76                                               Character count to wrap at.
-**mailType**        text                ``text`` or ``html``         Type of mail. If you send HTML email you must send it as a complete web
+**$wordWrap**       true                ``true``/``false``           Enable word-wrap.
+**$wrapChars**      76                                               Character count to wrap at.
+**$mailType**       text                ``text`` or ``html``         Type of mail. If you send HTML email you must send it as a complete web
```

Comment:
Please remove all `$` marks in `Preference`.
Because these items may be keys in an array.